docker/server/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.11-slim
+
+WORKDIR /app
+
+RUN pip install --upgrade pip && pip install poetry
+
+COPY ./server ./
+
+RUN poetry config virtualenvs.create false
+
+RUN poetry install
+
+COPY ./docker/server/entrypoint.sh /entrypoint.sh
+
+RUN chmod +x /entrypoint.sh
```

Comment:
after this step, you can also set the entrypoint script, so it's the default behaviour of the container instead of needing to specify it in the compose file